tf_encrypted/tensor/native.py

```diff
@@ -98,7 +98,7 @@ def sample_uniform(self,
       if secure_random.supports_secure_randomness():
         sampler = secure_random.random_uniform
       else:
-        sampler = tf.random_uniform
+        sampler = tf.random.uniform
```

Comment:
when will the random values be sampled? when this line is executed or during session run?

Reply:
Pretty sure it's when the tf.function is actually called. Unless you are running in eager mode but I'm not sure we want to provide any security guarentees for eager mode, for now anyway.

Reply:
> Unless you are running in eager mode

do we disable eager mode anywhere?

Reply:
Realized there aren't any uses of tf.function in this PR but I believe it should be the same for session runs.

Eager isn't explicitly disabled anywhere but it doesn't just work.